pkg/kritis/review/validating_transport_test.go

```diff
@@ -38,8 +39,8 @@ func encodeB64(in string) string {
 func TestValidatingTransport(t *testing.T) {
```

Comment:
I'm a bit concerned that this test suite has grown to such a size that it's hard to follow now. Maybe we should add type-specific testing elsewhere so this suite can focus more on the main logic?

Reply:
Done in a follow-up PR.